### PR TITLE
Fibonocci code is updated

### PR DESCRIPTION
**Pull request by dineshN52**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -1,7 +1,17 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+         public static void Main (string[] args) {
+            int n1 = 0, n2 = 1, n3, i, number;
```
Comment (ravin-imm) [inline Program.cs:4]: Variable "i" can be initialized in _for_ loop itself.  Try to add simple comments to what each variable stores.

File: Program.cs [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
-        }
-    }
+   internal class Program {
+         public static void Main (string[] args) {
+            int n1 = 0, n2 = 1, n3, i, number;
+            Console.Write ("Enter the number of elements: ");
+            number = int.Parse (Console.ReadLine ());
+            Console.Write (n1 + " " + n2 + " "); //printing 0 and 1    
+            for (i = 2; i < number; ++i) //loop starts from 2 because 0 and 1 are already printed    
+            {
```
Comment (ravin-imm) [inline Program.cs:9]: No need to leave a separate line for "{" braces. I think VS 2022 is not configured. Please configure it using the settings that was shared earlier.

**Review by ravin-imm — CHANGES_REQUESTED** [review]

File: Program.cs (hunk 1 above)
Comment (dineshN52) [reply Program.cs]: changed the initialization and updated sir

File: Program.cs (hunk 2 above)
Comment (dineshN52) [reply Program.cs]: VS is configured and change requested is performed and updated sir

File: Program.cs [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+namespace Training {
+   internal class Program {
+      public static void Main (string[] args) {
+         int n1 = 0, n2 = 1, n3, number; //n1 & n2 are input, n3 and number are temporary variable
```
Comment (ravin-imm) [inline Program.cs:5]: "n3 and number are temporary variables" is not a meaningful comment. It doesn't describe what they hold.

**Review by ravin-imm — CHANGES_REQUESTED** [review]

**Review by ravin-imm — APPROVED** [review]

**Pull request closed without merging**